src/System.Runtime.Extensions/tests/Performance/Perf.Environment.cs

```diff
@@ -65,5 +66,105 @@ public void ExpandEnvironmentVariables()
                 Environment.SetEnvironmentVariable(env, null);
             }
         }
+
+        [Benchmark]
+        public void GetEnvironmentVariables()
+        {
+            PerfUtils utils = new PerfUtils();
+            string env = utils.CreateString(15);
+            try
+            {
+                // setup the environment variable so we can read it
+                Environment.SetEnvironmentVariable(env, "value");
+
```

Comment:
@brianrob @DrewScoggins The warmup iteration is discarded when the API generates the statistics, and when the raw data is parsed by the BenchView tools.